app/assets/javascripts/connect/views/MapMiniControlsView.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * The MapMiniControlsView view.
+ *
+ * @return MapMiniControlsView view (extends Backbone.View)
+ */
+define([
+  'underscore',
+  'handlebars',
+  'mps',
+], function(_, Handlebars, mps) {
+
+  'use strict';
+
+  var MapMiniControlsView = Backbone.View.extend({
+
+    status: new (Backbone.Model.extend({
+      defaults: {
+        is_drawing: false
+      }
+    })),
+
+    el: '#map-controls',
+
+    events: {
+      'click .js-map-controls-zoom-in' : 'onClickZoomIn',
+      'click .js-map-controls-zoom-out' : 'onClickZoomOut',
+      'click .js-map-controls-autolocate' : 'onClickLocate',
+      'click .js-map-controls-drawing' : 'onClickDrawing',
+      // 'click .js-map-controls-zoom-search' : 'onClickZoomOut',
+    },
+
+    initialize: function(map) {
+      this.map = map;
+      this.cache();
+      this.listeners();
+
+      this.renderGoogleAutocomplete();
+    },
+
+    listeners: function() {
+      this.status.on('change:is_drawing', this.changeIsDrawing.bind(this));
```

Comment:
I didn't know that... Thank you!!! As long as I'm still working on this feature and it's not related to the PR, could we merge without changing this right now?